### PR TITLE
Release/v3.13.5

### PR DESCRIPTION
**Pull request by mathieulemieux**
BugFixes

- Remove commitizen and cz-conventional-changelog dependencies
- Change default graphkb_admin user for pori_admin

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,15074 +1,8 @@
 {
   "name": "@bcgsc-pori/graphkb-api",
-  "version": "3.13.4",
-  "lockfileVersion": 2,
+  "version": "3.13.5",
+  "lockfileVersion": 1,
```
Comment (creisle) [inline package-lock.json:4]: The lockfileversion should not have been downgraded. Did you use an older version of node here?

**Review by creisle — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (mathieulemieux) [reply package-lock.json]: Don't know why it changed, relly. I can't see myself knowingly using an older version of npm. Anyway, I reinstalled everything with npm 8.1.0, then runned into the same issue as here with the graphkb_loader https://github.com/bcgsc/pori_graphkb_loader/pull/102#issuecomment-1159134622 so fixed that too.

**Review by elewis2 — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (creisle) [reply package-lock.json]: Still says lockfileVersion 1. It should be 2, I think you will need to revert these commits and re-try ensuring you are using node 16

**Review by creisle — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (mathieulemieux) [reply package-lock.json]: It's not. It is lockfileVersion 2 now; where do you see lockfileVersion 1?

Comment (creisle) [reply package-lock.json]: weird, sorry about that. its showing as 1 on the diff in github. Perhaps that's a error in the display due to the size of the diff. I will checkout the branch and confirm it there

Comment (creisle) [reply package-lock.json]: you're right. manually looks ok

```git diff master..release/v3.13.5```

**Review by creisle — APPROVED** [review]

**Comment by mathieulemieux** [comment]
This PR will be closed without merge and the related branch https://github.com/bcgsc/pori_graphkb_api/tree/release/v3.13.5 will be deleted, in favor of a v3.14.0 release branch and PR.

**Pull request closed without merging**